app/frontend/src/search/hits.js

```diff
@@ -52,16 +52,28 @@ export const getJobAlertLinkParam = (key, value) => {
   return encodeURIComponent(`search_criteria[${key}]`) + `=${value.replace(' ', '+')}&`;
 };
 
-export const createHeadingMarkup = (numberHits, keyword = null, location = null) => {
-  const prefix = keyword || location ? ` ${numberHits > 1 ? 'match' : 'matches'} ` : ' listed';
-  const postfix = `${prefix}${createHeadingHTMLForSearchTerm('', keyword)} ${createHeadingHTMLForSearchTerm('near', location)}`;
-  const hits = keyword || location ? `<span class="govuk-!-font-weight-bold">${numberHits}</span>` : `There ${numberHits > 1 ? 'are' : 'is'} <span class="govuk-!-font-weight-bold">${numberHits}</span>`;
+export const createHeadingMarkup = (options) => {
+  const { count, keyword = '', location = '' } = options;
 
-  return `${hits} ${numberHits > 1 ? 'jobs' : 'job'} ${postfix}`;
+  const postfix = `${getSearchTermsPrefix(location, keyword, count)}${createHeadingHTMLForSearchTerm('', keyword, false)} ${createHeadingHTMLForSearchTerm('near', location, true)}`;
```

Comment:
I'm a bit confused about the use of 'postfix' and 'prefix' here and in the method getSearchTermsPrefix. Are they both referring to the same thing?

Reply:
yeah i might have got these a bit topsy turvy due to refactoring, ill get it a bit more understanbable

Reply:
have renamed variables in this function to be a bit intuitive i hope